tests/Models/User.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Propaganistas\LaravelPhone\Tests\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Propaganistas\LaravelPhone\Models\PhoneNumberCast;
+
+class User extends Model
+{
+    protected $fillable = [
+        'contact_number',
+        'contact_number_country',
+        'emergency_number',
+        'emergency_number_country',
+    ];
+
+    protected $casts = [
+        'contact_number' => PhoneNumberCast::class . ':contact_number_country',
+        'emergency_number' => PhoneNumberCast::class . ':emergency_number_country',
```

Comment:
Can you add a test in which the cast parameters include both a country field and several hardcoded countries?

Reply:
Yup, will be righting more test over the weekend :smile: